packages/core/src/node/runtimeModule/siteData/index.ts

```diff
@@ -118,16 +118,16 @@ export async function siteDataVMPlugin(context: FactoryContext) {
     locales: userConfig?.locales || userConfig.themeConfig?.locales || [],
     logo: userConfig?.logo || '',
     logoText: userConfig?.logoText || '',
-    ssg: userConfig?.ssg ?? true,
+    ssg: Boolean(userConfig?.ssg) ?? true,
     multiVersion: {
       default: userConfig?.multiVersion?.default || '',
       versions: userConfig?.multiVersion?.versions || [],
     },
     search: tempSearchObj ?? { mode: 'local' },
     pages: pages.map(page => {
-      const { content, id, domain, _filepath, ...rest } = page;
-      // In production, we cannot expose the complete filepath for security reasons
-      return isProduction() ? rest : { ...rest, _filepath };
+      // omit some fields for runtime size
+      const { content, id, domain, _filepath, _html, ...rest } = page;
```

Comment:
`_filepath` will not be exposed in both dev and production mode now?

Reply:
yes, it was very hacky before 😅